ogcapi-stable/ogcapi-html/src/main/java/de/ii/ogcapi/html/domain/MapClient.java

```diff
@@ -49,8 +49,17 @@ default boolean getSavePosition() {
     return false;
   }
 
+  List<Double> getCenter();
+
+  Optional<Number> getZoom();
+
   Optional<Map<String, String>> getBounds();
 
+  @Value.Default
+  default boolean getUseBounds() {
```

Comment:
I think this is redundant and can be removed. I do not see a case in the code where it would make any difference. Even if that was the case, getBounds is already optional, so just do not set it if you do not want bounds.

Reply:
It is necessary. If bounds are included in _map, the JavaScript always moves the view to the bounds. This is desired for Features (useBounds=true), but not for Tiles (useBounds=false), so the bounds are suppressed for tiles, if we have a style or explicit center information. See mapClient.mustache.